ChangeLog.d/add_final_delay_accessor

```diff
@@ -0,0 +1,3 @@
+Features
+   * Add a function to access the private final delay field in an
```

Comment:
Please give the name of the function so that interested users can find it easily.